app-profile-jee-html5/src/test/java/org/keycloak/quickstart/ArquillianProfileJeeHtml5Test.java

```diff
@@ -122,18 +121,26 @@ public static void cleanUp() throws IOException{
     }
 
     @Before
-    public void setup() {
+    public void setup() throws InterruptedException {
         webDriver.manage().timeouts().pageLoadTimeout(30, TimeUnit.SECONDS);
```

Comment:
@mhajas During further GH Actions testing I've found a test instability. Could you please add the following line which should fix it?
```java
webDriver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
```